client/src/Movies/Movie.js

```diff
@@ -18,7 +18,7 @@ const Movie = (props) => {
           console.error(error);
         });
 
-  },[]);
+  },[props.match.params]);
   
   // Uncomment this only when you have moved on to the stretch goals
```

Comment:
const saveMovie = () => {
  const addToSavedList = props.addToSavedList;	
   addToSavedList(movie)
}

{/**
You basically have the saved list built out all you need to do is uncomment this code and then add in an onClick handler on the button down below!!
 hook it up! 👓🚀☄🔧🛠

*/}